msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsg.jsp

```diff
@@ -59,9 +59,10 @@
 	  	                
 		  <h:column>
 		    <f:facet name="header">
-					<h:selectBooleanCheckbox id="checkAll" title="#{msgs.cdfm_checkall}">
-						<h:outputText value="#{msgs.cdfm_checkall} "/>
-					</h:selectBooleanCheckbox>
+				<h:panelGroup>
+					<h:selectBooleanCheckbox id="checkAll" title="#{msgs.cdfm_checkall}"/>
+					<h:outputText value="#{msgs.cdfm_checkall} "/>
```

Comment:
Sorry to complicate, but we should probably get this right while we are editing: shouldn't this outputText be an outputLabel instead?

Reply:
It's totally fine @ottenhoff ! I've done the changes, thanks for the suggestions.